src/flb_parser_json.c

```diff
@@ -165,7 +165,7 @@ int flb_parser_json_do(struct flb_parser *parser,
         tmp[len] = '\0';
         flb_warn("[parser:%s] Invalid time format %s for '%s'.",
                  parser->name, parser->time_fmt, tmp);
-        time_lookup = time(NULL);
+        time_lookup = 0;
```

Comment:
what are the side effects of this change for a non-kubernetes use case ?

Reply:
I didn't find side effects when checking the code. Anyway, falling back to current time should not be the responsibility of the parser as there is no way to distinguish a fallback from a successful time parsing. This should be the responsibility of the calling code.

Reply:
looking at the code flb_parser_regex, it's always assumed if time lookup resolution fails, assign the current timestamp. Let's preserve that behavior.